assembl/models/attachment.py

```diff
@@ -418,3 +419,35 @@ class AgentProfileAttachment(Attachment):
 
     def is_owner(self, user_id):
         return user_id == self.user_id
+        
+class VoteSessionAttachment(Attachment):
+
+    __tablename__ = "vote_session_attachment"
+
+    id = Column(Integer, ForeignKey(
+        'attachment.id',
+        ondelete='CASCADE',
+        onupdate='CASCADE'
+    ), primary_key=True)
+
+    vote_session_id = Column(Integer, ForeignKey(
+        'vote_session.id',
+        ondelete='CASCADE',
+        onupdate='CASCADE',
+        ),
+        nullable=False,
+        index=True)
+
+    vote_session = relationship(
+        VoteSession,
+        backref=backref(
+            'attachments',
+            cascade="all, delete-orphan"),
+    )
+    __mapper_args__ = {
+        'polymorphic_identity': 'vote_session_attachment',
+        'with_polymorphic': '*'
+    }
+
+    # TODO: permissions !!!
```

Comment:
still to do ?

Reply:
yep

Reply:
but we decided to not do it on this branch